python/do_username/generate.py

```diff
@@ -0,0 +1,34 @@
+import math
+import random
+
+sea_creatures = "walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl".split(" ")
+sea_objects = "boat ship submarine yacht dinghy raft kelp seaweed anchor".split(" ")
+adjective_descriptors = "cute adorable lovable happy sandy bubbly friendly floating drifting".split(" ")
+size_descriptors = "large big small giant massive tiny little yuge".split(" ")
```

Comment:
```suggestion
SIZE_DESCRIPTORS = tuple("large big small giant massive tiny little".split(" "))
```
Uppercase for consistency, this is a constant. Cast to tuple so that it's immutable. Remove `yuge`, not in Ruby version.